dev/dll/XamlControlsResources.cpp

```diff
@@ -173,7 +173,7 @@ void SetDefaultStyleKeyWorker(winrt::IControlProtected const& controlProtected,
     if (auto control5 = controlProtected.try_as<winrt::IControl5>())
     {
         winrt::Uri uri{
-            []() -> wstring_view {
+            []() -> std::wstring {
```

Comment:
Explicit "-> return type" should only be needed when it can't be inferred by the return statement in the lambda. But here we have just one return so it should figure it out... right?

Reply:
yes. no matter wstring, wstring_view or remove it, it passed the compilation locally.

Reply:
removed